plugin/src/shared/generators/form-utils/generator.ts

```diff
@@ -0,0 +1,23 @@
+import { execSync } from 'child_process';
+import * as path from 'path';
+import { formatFiles, generateFiles, Tree } from '@nx/devkit';
+
+export async function runFormUtilsGenerator(
+  tree: Tree,
+  options: { name: string; directory: string }
+) {
+  const libRoot = `libs/${options.directory}`;
+
+  // Generate shared libs
+  execSync(`npx nx g react-lib ${options.directory}/shared/utils/form`, { stdio: 'inherit' });
```

Comment:
could you please update the command args? The option `directory` was removed in https://github.com/RonasIT/nx-generators/pull/37. You can use `app=shared` option instead.